_posts/2023-03-01-mirrors.md

```diff
@@ -70,15 +70,15 @@ With the recent progress on [realtime ray tracing](https://en.wikipedia.org/wiki
+However, let's start by saying that there's also a very *wrong* way of doing it, and I've seen it implemented on some of the most expensive driving simulator out there, so I think it deserves a warning. If you have a continuous 360° display, **don't** just use real actual mirrors that reflect the display. Mirrors' images should be computed from a point of view that is **not** the "direct" view. The only valid application of real physical mirror is if they point toward a display area that is generated from the *mirrored* driver point of view, and not their *actual* point of view.
 
-<blockquote class="twitter-tweet"><p lang="en" dir="ltr">We&#39;ve completed the <a href="https://twitter.com/hashtag/nDisplay?src=hash&amp;ref_src=twsrc%5Etfw">#nDisplay</a> configuration of our largest <a href="https://twitter.com/hashtag/DrivingSimulation?src=hash&amp;ref_src=twsrc%5Etfw">#DrivingSimulation</a> platform at <a href="https://twitter.com/LescotUniEiffel?ref_src=twsrc%5Etfw">@LescotUniEiffel</a>. 12 displays (including tricky mirrors) running on 8 computers. Easy to setup and free-of-charge thanks to <a href="https://twitter.com/hashtag/UE4?src=hash&amp;ref_src=twsrc%5Etfw">#UE4</a>!<br><br>Let&#39;s do some <a href="https://twitter.com/hashtag/mobility?src=hash&amp;ref_src=twsrc%5Etfw">#mobility</a> <a href="https://twitter.com/hashtag/research?src=hash&amp;ref_src=twsrc%5Etfw">#research</a> now! <a href="https://t.co/FnsmzSlqJT">pic.twitter.com/FnsmzSlqJT</a></p>&mdash; Bertrand Richard (@brifsttar) <a href="https://twitter.com/brifsttar/status/1458740786481307658?ref_src=twsrc%5Etfw">November 11, 2021</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+<blockquote class="twitter-tweet"><p lang="en" dir="ltr">We&#39;ve completed the <a href="https://twitter.com/hashtag/nDisplay?src=hash&amp;ref_src=twsrc%5Etfw">#nDisplay</a> configuration of our largest <a href="https://twitter.com/hashtag/DrivingSimulation?src=hash&amp;ref_src=twsrc%5Etfw">#DrivingSimulation</a> platform at <a href="https://twitter.com/LescotUniEiffel?ref_src=twsrc%5Etfw">@LescotUniEiffel</a>. 12 displays (including tricky mirrors) running on 8 computers. Easy to set up and free-of-charge thanks to <a href="https://twitter.com/hashtag/UE4?src=hash&amp;ref_src=twsrc%5Etfw">#UE4</a>!<br><br>Let&#39;s do some <a href="https://twitter.com/hashtag/mobility?src=hash&amp;ref_src=twsrc%5Etfw">#mobility</a> <a href="https://twitter.com/hashtag/research?src=hash&amp;ref_src=twsrc%5Etfw">#research</a> now! <a href="https://t.co/FnsmzSlqJT">pic.twitter.com/FnsmzSlqJT</a></p>&mdash; Bertrand Richard (@brifsttar) <a href="https://twitter.com/brifsttar/status/1458740786481307658?ref_src=twsrc%5Etfw">November 11, 2021</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
```

Comment:
What changed here?

Reply:
But it's in an embedded tweet though, I think it might be better to leave the typo since it's in a the Tweet anyway.

Reply:
Indeed, I did not realize it was in a tweet